Airport Ticket Booking/Passenger.cs

```diff
@@ -0,0 +1,272 @@
+            if (IsBooking)
+            {
+                Console.WriteLine("Choose from the following what do you want to modify: ");
+                Console.WriteLine("1. Modify the Passenger Name");
+                Console.WriteLine("2. Modify the Flight Id");
+                Console.WriteLine("3. Modify The Class");
+                Console.WriteLine("4. Modify all");
+
+                int number = (int)UserInput((int)IntOrDouble.integern, 1, 4);
+                string newPassName = null;
+                int NewflightId = 1; // By the end the id will change it is just initial value
+                int NewFlightClass = (int) 1;
+                switch (number)
+                {
+                    case 1:
+                        Console.WriteLine("Please Enter the New Passenger Name:");
+                        newPassName = Console.ReadLine();
+                        break;
+                    case 2:
+                        Console.WriteLine("Please Enter the New Flight Id:");
+                        NewflightId = (int) UserInput((int)IntOrDouble.integern);
+                        break;
+                    case 3:
+                        Console.WriteLine("Please Enter the New Class:");
+                        Console.WriteLine("1 : Economy");
+                        Console.WriteLine("2 : Business");
+                        Console.WriteLine("3 : FirstClass");
+                        NewFlightClass = (int)UserInput((int)IntOrDouble.integern, 1, 3);
+                        break;
+                    case 4:
+                        Console.WriteLine("Please Enter the New Name:");
+                        newPassName = Console.ReadLine();
+
+                        Console.WriteLine("Please Enter the New Flight Id:");
+                        NewflightId = (int)UserInput((int)IntOrDouble.integern);
+
+                        Console.WriteLine("Please Enter the New Class:");
+                        Console.WriteLine("1 : Economy");
+                        Console.WriteLine("2 : Business");
+                        Console.WriteLine("3 : FirstClass");
+                        NewFlightClass = (int)UserInput((int)IntOrDouble.integern, 1, 3);
+                        break;
+                    default:
+                        break;
+                } // end switch
+                FlightsManager.ModifyBooking(number, id, newPassName, NewflightId, NewFlightClass);
+            }
+            else
+            {
+                Console.WriteLine("There is no Booking with this Id!");
+            }
+        }
+
+        private void BookAskInput()
+        {
+            Console.WriteLine("Please Enter Your Name: ");
+            string PassengerName = Console.ReadLine();
+
+            Console.WriteLine("Please Enter the Id For your Flight: ");
+            int FlightId = (int) UserInput((int) IntOrDouble.integern);
+            while (!FlightsManager.isInFlights(FlightId))
```

Comment:
is it possible to optimize this? this is going to be O(n^2) which's quite not efficient, specially when the flights list is large